spec/spec_helper.rb

```diff
@@ -42,22 +40,6 @@
   Kernel.srand config.seed
 
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-  config.use_transactional_fixtures = false
+  config.use_transactional_fixtures = true
   config.infer_spec_type_from_file_location!
-
-  config.before(:each) do
-    User.create!(permissions: ["signin", "GDS Editor"])
-    DatabaseCleaner.start
-    WebMock.reset!
```

Comment:
Shouldn't `User.create!` and `WebMock.reset!` still exist here? I'm trying to work out if/why they're not needed any more.

I was expecting for this commit to only remove things relating to `DatabaseCleaner`.

Reply:
Ah yes, sorry I should have covered those details.

I've amended this commit now to explain. 

Essentially we don't need the user because the db:seeds create it (and we certainly didn't need one for every test 😢).

Webmock.reset! is done automatically by Webmock so we don't need to call it ourselves.